test/validate/plant_parts_eia_test.py

```diff
@@ -22,61 +24,58 @@
 #################
 
 
-def test_run_aggregations(pudl_out_eia, live_dbs):
+def test_run_aggregations(live_dbs: bool, pudl_engine: sa.Engine) -> None:
     """Run a test of the aggregated columns.
 
     This test will used the plant_parts_eia, re-run groubys and check similarity.
     """
     if not live_dbs:
         pytest.skip("Data validation only works with a live PUDL DB.")
-    if pudl_out_eia.freq == "YS":  # Annual only.
-        logger.info("Testing ownership fractions for owned records.")
```

Comment:
Most of the lines changed below are just because I flipped the nature of this check, so it bails early.